Bottom: fe69bd2ffe5c8fda95ffac3a92ded842316ebc2b
Top:    069839e66b4bd2a0f7dc42bcb6308e3c00a47d41
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 17:31:40 +0000

IMPORTANT: DONT DRAW POUR OUTLINES DURING EXPORT

They can produce shorted boards otherwise!


---

diff --git a/src/draw.c b/src/draw.c
index 49f011f..1c1a504 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
